test/musicbrainz_db/scripts/createdb.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
These files are copied from the musicbrainz-docker project. Eventually we should be able to merge these projects together to remove the code duplication, but for now this is the easiest way of getting a sample database dump loaded into the `musicbrainz-test-database` image